client/src/components/Header.js

```diff
@@ -5,151 +5,142 @@ import React, { useState } from "react";
-            >
-              Rent
-            </NavLink>
-          </Grid>
-          {auth ? (
-            <Grid>
-              <IconButton
-                aria-label="account of current user"
-                aria-controls="menu-appbar"
-                aria-haspopup="true"
-                onClick={handleMenu}
-                color="inherit"
-              >
-                <Avatar
-                  className={classes.avatar}
-                  src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTdX7wWCMOvGYD6_4-MthVKf-DjjgLF_GqQzg&usqp=CAU"
-                />
-              </IconButton>
-              <Menu
-                id="menu-appbar"
-                anchorEl={anchorEl}
-                anchorOrigin={{
-                  vertical: "top",
-                  horizontal: "right",
-                }}
-                keepMounted
-                transformOrigin={{
-                  vertical: "top",
-                  horizontal: "right",
-                }}
-                open={open}
-                onClose={handleClose}
-              >
-                <MenuItem onClick={handleClose}>Profile</MenuItem>
-                <MenuItem onClick={handleClose}>Logout</MenuItem>
-              </Menu>
-            </Grid>
-          ) : (
-            <div>
-              <Button onClick={openModal} className={classes.loginbtn}>
-                Login
-              </Button>
-              <Modal
-                aria-labelledby="modal-title"
-                className={classes.modal}
-                open={modal}
-                onClose={closeModal}
-              >
-                {signup ? (
-                  <WrappedSignup close={closeModal} signin={handleSignin} />
-                ) : (
-                  <WrappedSignin close={closeModal} signup={handleSignup} />
-                )}
-              </Modal>
-            </div>
-          )}
-          <Cart />
-        </Toolbar>
-      </AppBar>
-      <Toolbar />
-    </Grid>
-  );
+	const WrappedSignin = React.forwardRef((props, ref) => (
+		<Signin {...props} forwardedRef={ref} />
+	));
+	const WrappedSignup = React.forwardRef((props, ref) => (
+		<Signup {...props} forwardedRef={ref} />
+	));
+
+	return (
+		<Grid container className={classes.root}>
+			<AppBar position="fixed" className={classes.appbar}>
+				<Toolbar>
+					<IconButton
+						edge="start"
+						className={classes.menuButton}
+						color="inherit"
+						aria-label="menu"
+					/>
+					<Typography variant="h6" className={classes.title}>
+						Travel Booking
+					</Typography>
+					<Grid className={classes.pages}>
+						<NavLink className={classes.navlinks} to="/explore">
+							Explore
+						</NavLink>
+						<NavLink className={classes.navlinks} to="/flights">
+							Flights
+						</NavLink>
+						<NavLink className={classes.navlinks} to="/hotel">
+							Hotels
+						</NavLink>
+						<NavLink className={classes.navlinks} to="/rent">
+							Rent
+						</NavLink>
```

Comment:
can we actually change this to `Cars`, the mockup says Rent but that doesn't make sense lol